src/garage/envs/wrappers/multi_env_wrapper.py

```diff
@@ -0,0 +1,151 @@
+"""
+A wrapper env that handles multiple tasks from different envs.
+
+Useful while training multi-task reinforcement learning algorithms.
+It provides observations augmented with one-hot representation of tasks.
+"""
+
+import random
+
+import akro
+import gym
+import numpy as np
+
+
+class SampleStrategy:
+    """Base sample stretegy class.
+
+    Args:
+        num_tasks (int): Total number of tasks.
+    """
+
+    def __init__(self, num_tasks):
+        self._num_tasks = num_tasks
+
+    def sample_tasks(self):
+        """Sample new task."""
+        pass
+
+
+class RoundRobinStrategy(SampleStrategy):
+    """A sample stretegy class for sampling in round robin fashion."""
+
+    def __init__(self, num_tasks):
+        super().__init__(num_tasks)
+        self._last_task = None
+
+    def sample_tasks(self):
+        """Sample new task."""
+        if self._last_task is None:
+            self._last_task = 0
+            return 0
+
+        new_task = (self._last_task + 1) % self._num_tasks
+        self._last_task = new_task
+        return new_task
+
+
+class UniformRandomStrategy(SampleStrategy):
+    """A sample stretegy class for sampling uniformly at random."""
+
+    def sample_tasks(self):
```

Comment:
This is saying sample tasks so I think it should be able to sample more than one tasks?

Reply:
Its a typo. It should only sample one task.

Reply:
We should probably rename this to `sample_task`.